code/__HELPERS/maths.dm

```diff
@@ -85,3 +85,18 @@ var/gaussian_next
 		gaussian_next = R2 * working
 	return (mean + stddev * R1)
 #undef ACCURACY
+
+
+
+// oof, what a mouthful
+// Used in status_procs' "adjust" to let them modify a status effect by a given
+// amount, without inadverdently increasing it in the wrong direction
+/proc/directional_bounded_sum(orig_val, modifier, bound_lower, bound_upper)

```

Comment:
Could you uh, explain why this is necessary? It seems like a case of "coders are too dumb to use a proc properly, so let's idiot proof it", but I may be completely misunderstanding the intent of this proc.